### ISSUE TITLE
documentation and tutorial with meaningful use case

### ISSUE THREAD

**Issue by vsoch** [poldracklab/cogat#124]
heyo! I'm putting together a (interactive video, probably to @poldrack dislike :P) for Cognitive Atlas, and I think it would actually first be better to have a Github Pages sort of "Getting Started" guide. I say Github Pages and not the actual site because it would be something that could be easily added to a README. I'm thinking something along the lines of walking through a use case, like:

 - I am a graduate student interested in X
 - I come to the cognitive atlas to search for Y
 - When I find Y I do ... click on... etc

TLDR: I started with X, discovered Y, then was taken to this other tool, oh, and by the way the whole thing is do-able via a programmatic API .... here's a notebook for that!

I would be down to put this together, but I'm pretty terrible with actual (useful / meaningful) use cases so I'd ask for help / wisdom from some of Poldracklab for this! What do you think?

**Comment by vsoch** [comment]
hey @rwblair I'm guessing there isn't bandwidth for this, I put together a notebook in case you have any feedback, at least of how I thought about use cases for the cognitive atlas https://github.com/vsoch/dinosaur-lessons/blob/master/neuro/notebook/cognitive-atlas-intro.ipynb For the neurosynth part of the analysis, [we discussed](https://github.com/neurosynth/neurosynth/issues/81#issuecomment-414028951) that really the best way to interact with the tool is the interface, and then wait for NiMare (mraraaaahahaha) to come out for programmatic stuffs. if you don't have any feedback, I'm happy to close the issue.
